prow/spyglass/lenses/metadata/template.html

```diff
@@ -8,6 +8,8 @@
 <p class="test-summary">Test started <abbr id="summary-start-time" title="{{.StartTime}}">{{.StartTime}}</abbr> {{if .Finished -}}
 {{- if .Passed -}}
   <span class="passed">passed</span>
+{{- else if .Errored -}}
+  <span class="failed">error</span>
```

Comment:
nit: This doesn't seem grammatically consistent, but I would have put `errored` and apparently that isn't even a word 🙃

Reply:
@cjwagner fair enough :) I am hoping to circle back and hopefully surface additional information about why a scheduling timeout happens, but there isn't much surfaced on the `prowjob.json` at this point so it'll need some work